driver-embedded/src/test/functional/com/mongodb/embedded/client/MongoClientsTest.java

```diff
@@ -41,15 +49,15 @@ public void shouldThrowIfCreateCalledWithoutCallingInit() {
     }
 
     @Test(expected = MongoClientException.class)
-    public void shouldThrowIfUsingMongoClientAfterClose() {
+    public void shouldThrowIfLibHandlesStillExist() {
+        MongoClients.init(getMongoEmbeddedSettings());
         MongoClient mongoClient = MongoClients.create(getMongoClientSettings());
         try {
             MongoClients.close();
```

Comment:
Add a `fail()` at the end of the try block to ensure that it's the call within the block that's the one that's actually throwing, rather than either of the ones in the finally block

Reply:
Done